azure-devops/azext_devops/dev/common/credential_store.py

```diff
@@ -79,9 +82,11 @@ def clear_password(self, key):
             keyring.delete_password(key, self._USERNAME)
```

Comment:
may be a corner case, but what if set has issues and delete does not have issue.
In that case customer will always see `The credential was not found`.

Should we instead look in both places and clean?

Reply:
Will check in both keyring and file for
1. get password
2. delete password